tests/unit/test_mips.py

```diff
@@ -9,8 +10,9 @@
 
 def test_mips(mocker, requests_mock):
```

Comment:
there seems to be multiple tests in this one test function.  can we please split this test into multiple tests?

Reply:
sure I can break this up into smaller tests